azure-pipelines.yml

```diff
@@ -0,0 +1,96 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+variables:
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+
+jobs:
```

Comment:
We can have a single job on the DAS - Continuous Integration pool, with a demand to be Windows like https://github.com/SkillsFundingAgency/das-alpha-spikes/blob/master/FATv2/APIM/SFA.DAS.FATv2.Alpha/azure-pipelines.yml#L20-L22 - a recent change

Reply:
Refactored to use this,